app/graphql/types/repository_connection_with_total_type.rb

```diff
@@ -5,12 +5,10 @@ class RepositoryConnectionWithTotalType < BaseConnection
   field_class GraphQL::Cache::Field
 
   field :total_count, Integer, null: true, cache: true
-  field :years, [FacetType], null: true, cache: true
-  field :members, [FacetType], null: true, cache: true
```

Comment:
Do we need to remove the Member facet functionality from this interface? I mean this information does still exist for all the repos that have datacite dois.

Reply:
implemented